Coins: update manual labeling (#1437)
zkSNACKs/WasabiDoc: docs/why-wasabi/Coins.md

### Manual coin labeling and selection

Using Wasabi it is mandatory to label every receiving and destination address.

> This helps the user to know where your coins came from so that you can judge whether there are privacy concerns when sending a specific coin to a specific receiver.

Reviewer: I guess since currently the auto coin selection is the default you should be mentioning that the labels help the wallet select the best combination of coins.

```suggestion
This helps the user to know where their coins came from so that they can judge whether there are privacy concerns when sending a specific coin to a specific receiver.
```

Reviewer: > In Wasabi it is mandatory to give every receive address a label of those who know that this address is yours, so that later you know what to do with it, and it also helps the auto coin selection algorithm when sending.

Something like the above mentioned in https://docs.wasabiwallet.io/using-wasabi/Receive.html#the-importance-of-labeling
